site/src/_pages/spec/1_block-types.mdx

```diff
@@ -523,6 +619,4 @@ This could be
 
 Blocks SHOULD provide at least basic visual styling to allow them to be embedded and used without modification by any web application.
 
-Blocks SHOULD use the CSS variables listed in [Appendix A](https://blockprotocol.org/spec/appendix-a-styling) as property values where appropriate, but SHOULD provide [fallback values](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties#custom_property_fallback_values) in case the embedding application does not define them.
-
-Ideally, the embedding application will also provide a `styleVariables` object alongside block data, which includes keys with the same names as the CSS variables listed in [Appendix A](https://blockprotocol.org/spec/appendix-a-styling) and appropriate values.
+Blocks SHOULD use the theme variables provided under the `styleVariables` object and listed in [Appendix A](https://blockprotocol.org/spec/appendix-a-styling) as property values where appropriate, but SHOULD provide [fallback values](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties#custom_property_fallback_values) in case the embedding application does not define them.
```

Comment:
```suggestion
Blocks SHOULD use the theme variables provided under the `styleVariables` object and listed in [Appendix A](https://blockprotocol.org/spec/appendix-a-styling) as property values where appropriate, and provide [fallback values](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties#custom_property_fallback_values) in case the embedding application does not define them.
```